packages/tools/pluggable-widgets-tools/scripts/e2e.js

```diff
@@ -130,6 +134,10 @@ async function exists(filePath) {
 
 async function getLatestMendixVersion() {
```

Comment:
I think it's better to rename this function to `getMendixVersion` as it doesn't necessarily return the latest version anymore

Reply:
Fixed